src/common/config_parser.cpp

```diff
@@ -44,8 +44,7 @@ void ConfigParser::addOptionsGeneral(cli::CLIWrapper& cli) {
 
   // clang-format off
   cli.add<bool>("--version",
-     "Print version number and exit",
-     false);
+     "Print version number and exit");
```

Comment:
Is this false by default?

Reply:
I removed the definition of --version from ConfigParser. Now it's automatically added like --help is and not present in `config`